ironfish-rust-nodejs/src/frost.rs

```diff
@@ -203,13 +203,19 @@ pub fn generate_and_split_key(
     let identities = try_deserialize_identities(identities)?;
 
     let packages =
-        split_spender_key(&spending_key, min_signers, identities).map_err(to_napi_err)?;
+        split_spender_key(&spending_key, min_signers, identities.clone()).map_err(to_napi_err)?;
```

Comment:
We should change all these functions that expect a Vec as an input to accept a slice or an iter, so that cloning can be avoided here and in other places

Reply:
I created a ticket for this: https://linear.app/if-labs/issue/IFL-2358/replace-vec-in-function-inputs-with-iter-or-arrays